index.rst

```diff
@@ -319,21 +319,14 @@ Documentation to be collected for the release notes in pipelines_lsst_io_ is:
 Documenting Deprecations
 """"""""""""""""""""""""
 
-Deprecations are divided into two groups:
-
-- **Pending Deprecations**: the methods and functions that are marked as
-deprecated and will be removed **after** the next major release is done. This
-implies they will still be available for some time.
-
-- **Actual Deprecations**: the methods and functions that will be removed
-**before** the next major release. This implies they will not be available in
-future releases.
+Deprecated interfaces may be removed in the major release after the one in which their deprecations first appear.
+These deprecations must be included in the release notes.
```

Comment:
While we won't have "actual deprecations" as defined previously, I think it's still good to have separate lists for the deprecations (deprecation warning in the code, may be removed in the next release) as well as the removals (code removed in this release) in the release notes.